test/sagemaker_tests/mxnet/inference/integration/local/local_mode_utils.py

```diff
@@ -18,7 +18,7 @@
 import tarfile
 import time
 
-from test.integration import RESOURCE_PATH
+from ...integration import RESOURCE_PATH
```

Comment:
Is it a good practice to use relative path for importing? Please also talk with other experienced engineers for it.

Reply:
I have used #210 as base. I can use direct path if it is more advisable.

Reply:
using absolute import was creating was creating a a longer path
from test/sagemaker_tests/trensorflow/tensorflow_training/integration import RESOURCE_PATH 

we can change to this in future if needed